src/Controller/Component/AuthComponent.php

```diff
@@ -537,9 +537,14 @@ public function getAuthorize($alias)
      *
      * You can use allow with either an array or a simple string.
      *
-     * `$this->Auth->allow('view');`
-     * `$this->Auth->allow(['edit', 'add']);`
-     * `$this->Auth->allow();` to allow all actions
+     * ```
+     * $this->Auth->allow('view');
+     * $this->Auth->allow(['edit', 'add']);
+     * ```
+     * or to allow all actions
```

Comment:
A new line above and below the code fence is not required? I don't remember myself.